types/mongodb/index.d.ts

```diff
@@ -864,8 +865,15 @@ export interface CollectionCreateOptions extends CommonOptions {
     pkFactory?: object;
     readPreference?: ReadPreferenceOrMode;
     serializeFunctions?: boolean;
+    /**
+     * @deprecated
+     * @see https://jira.mongodb.org/browse/NODE-2746
+     */
     strict?: boolean;
     capped?: boolean;
+    /**
+     * @deprecated
+     */
     autoIndexId?: boolean;
```

Comment:
```suggestion
```

This option was removed in MongoDB 3.4, and the typings are for 3.6

ref: https://stackoverflow.com/a/41783524/148072

Reply:
We should maybe keep this property to let the developers using this one know that is deprecated before removing it ?

Reply:
Giving a chance to make this discussion